packages/terra-list/src/ListItem.scss

```diff
@@ -1,30 +1,13 @@
 @import '~terra-mixins';
 @import './variables';
 
-.terra-ListItem-hasChevron {
-  display: block;
-  position: relative; // relative position so the chevron can use absolute position to vertically center itself
 
-  [dir='ltr'] & {
-    @include terra-padding-end(20px);
-  }
-
-  [dir='rtl'] & {
-    @include terra-padding-start(20px);
-  }
+.terra-ListItem-chevron {
+  @include terra-padding-end(10px);
```

Comment:
We should probably pad both the left and right side. The left side padding was to ensure there was white-space between text and the icon if the text filled the width of the content area.